runtime/src/lib.rs

```diff
@@ -87,7 +87,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     // implementation changes and behavior does not, then leave spec_version as
     // is and increment impl_version.
     spec_version: 229,
-    impl_version: 0,
+    impl_version: 1,
```

Comment:
I would argue that this changes the behavior (if a consumers application works now, it might stop working if we change fees).